_transactions.md

```diff
@@ -117,7 +117,7 @@ The first step is to prepare the transaction by specifying:
 - The _currency_ to denominate the transaction by.
 - The _amount_ of value to send in the denominated currency.
 - The _origin_ of the transaction, which can be an account id in the case of a _deposit_.
-- The _destination_ of the transaction, which can be in the form of a bitcoin address, an email address, an account id or an application id.
+- The _destination_ of the transaction, which can be in the form of a [crypto network address](#create-card-address), an email address, an account id, an application id, or a [card id](#card-object).
```

Comment:
```suggestion
- The _destination_ of the transaction, which can be in the form of a [crypto network address](_cards.md#create-card-address), an email address, an account id, an application id, or a [card id](_entities.md#card-object).
```

Shouldn't we reference the files in which these anchors are in? As it is now they won't lead anywhere.

Reply:
God question — this should probably be documented in the README. This linking method is related to how the docs are rendered by [Slate](https://github.com/slatedocs/slate): they are shown in a single-page website, so relative links like this will work (and absolute ones won't).

These two will point to https://uphold.com/en/developer/api/documentation/#create-card-address and https://uphold.com/en/developer/api/documentation/#card-object, respectively.